cve_bin_tool/extractor.py

```diff
@@ -67,15 +67,19 @@ def can_extract(self, filename):
     def extract_file_tar(cls, filename, extraction_path):
         """ Extract tar files """
         if not inpath("tar"):
-            print("Error: need 'tar' binary in path")
+            """ Acutally MinGW provides tar, so this might never get called """
+            shutil.unpack_archive(filename, extraction_path)
         else:
             return subprocess.call(["tar", "-C", extraction_path, "-axf", filename])
 
     @classmethod
     def extract_file_rpm(cls, filename, extraction_path):
         """ Extract rpm packages """
         if not inpath("rpm2cpio") or not inpath("cpio"):
-            print("Error: need 'rpm2cpio' or 'cpio' binary in path")
```

Comment:
I think that's a good plan. Also, I think we should be raising exceptions here. Since one can opt into extraction, they'd expect things to stop ship if its not extracting and they told it to. For instance in CI if someone turns this on they may not check the log unless it fails their job

Reply:
Hm, I was thinking just print to stderr instead of stdout, but I think you're right that this should be an exception.

Reply:
Looking better!  I think now it's not going to ask for 7z on windows, though (looks like everything's indented under line 79 below)